script.module.inputstreamhelper/test/kodi_six/__init__.py

```diff
@@ -0,0 +1,11 @@
+# coding: utf-8
```

Comment:
Can you also remove the test folder from the submission? It is not used by kodi too.